### PR TITLE
add yarn install instructions

### PR DESCRIPTION
**Pull request by weihai4099**
NOTE: Please read the Contributing.md guidelines before submitting your patch:

https://github.com/apache/incubator-apisix/blob/master/Contributing.md#how-to-add-a-new-feature-or-change-an-existing-one

### Summary

add one instruction

### Full changelog

* add yarn install instructions

### Issues resolved

Fix #XXX

### CODE REVIEW

File: README_CN.md [hunk 1]
```diff
@@ -143,7 +143,7 @@ APISIX 内置了对 Dashboard 的支持，使用步骤如下：
 ```
 git clone https://github.com/apache/incubator-apisix-dashboard.git
 ```
-
+- 安装yarn: 参考[安装文档](https://yarnpkg.com/zh-Hans/docs/install#windows-stable)
```
Comment (membphis) [inline README_CN.md]: default link: `https://yarnpkg.com/zh-Hans/docs/install`

Comment (weihai4099) [reply README_CN.md]: Thank you. Please check again

**Comment by membphis** [comment]
@weihai4099 We also need to update the English doc.

https://github.com/apache/incubator-apisix/blob/master/README.md#dashboard

**Comment by weihai4099** [comment]
everything all right

**Pull request merged**